transforms/module-exports-to-named-export.js

```diff
@@ -63,7 +63,7 @@ function transformer(file, api, options) {
         // module.export.b = a
         // → export { a as b }
         if (id.type === "Identifier" && init.type === "Identifier") {
-            return j.exportNamedDeclaration(null, [j.exportSpecifier(init, id)]);
+            return j.exportNamedDeclaration(null, [j.exportSpecifier.from({ exported: init, local: id })]);
```

Comment:
https://github.com/azu/commonjs-to-es-module-codemod/actions/runs/3859171360/jobs/6578501112
```
    - Expected  - 1
    + Received  + 1

    @@ -8,6 +8,6 @@

      const c = () => {
        return 42;
      };
      export { c };
    - export { c as d };
    + export { d as c };
```

The `exported` and `local` may be reversed.

Reply:
you are right - sorry about that!